Makefile

```diff
@@ -97,6 +98,14 @@ clean:
 		@rm -f README.md $(SRC)/circleci-base/Dockerfile
 		@$(MAKE) -C test clean
 
+format: format-sh
+
+format-sh:
+ifndef SHFMT
+$(error "shfmt is not installed: https://github.com/mvdan/sh")
```

Comment:
Might want to indent this so it doesn't complain if not found locally while executing other targets

Reply:
Fixed!